subgraphs/bar.graphql

```diff
@@ -0,0 +1,55 @@
+# Bar
+type Bar @entity {
+  # Contract address
+  id: ID!
+
+  # Decimals
+  decimals: Int!
+
+  # Name
+  name: String!
+
+  # Sushi token
+  sushi: Bytes!
+
+  # Symbol
+  symbol: String!
+
+  # Total xSushi supply
+  totalSupply: BigDecimal!
+
+  # Staked Sushi
+  staked: BigDecimal!
+
+  # Ratio xSushi:Sushi
```

Comment:
Switch this to sushi per xsushi. I think that makes more sense doing it that way.